lib/iris/fileformats/grib/load_rules.py

```diff
@@ -176,6 +176,11 @@ def add_bounded_time_coords(aux_coords_and_dims, grib):
 
     if \
             (grib.edition == 1) and \
+            (grib.timeRangeIndicator == 2):
+        add_bounded_time_coords(aux_coords_and_dims, grib)
+
```

Comment:
@lbdreyer is it easy to add a test in `tests/unit/fileformats/grib/load_rules/test_convert.py` within the `TestBoundedTime` class ... something along the lines of ...

```
    def test_time_range_indicator_3(self):
        self.assert_bounded_message(timeRangeIndicator=2)
```
Hopefully, this is relevant to your change ...